### ISSUE TITLE
create subquery always add at the end

### ISSUE THREAD

**Issue by NicolasAnquetil** [moosetechnology/FamixQueries#33]
to reproduce:
- create two top queries (1 and 2)
- create a subquery of the first one
- it will appear as a sub query of the second one (at the bottom of the list)

The sub query should be inserted as a child of the query that created it

**Comment by jordanmontt** [comment]
Yes you are right. The query it will be shown at the bottom of the list but is not a child of the second one. Is a child of the first one. This a problem only of the visualisation because the created queries always go to the bottom of the list.
We discussed this when I was in Bolivia and you told me to do it like that: to put the queries always to the bottom of the list. But I agree that this is confusing and that we have to change it
